modules/core/04-channel/keeper/upgrade.go

```diff
@@ -308,6 +308,38 @@ func (k Keeper) startFlushUpgradeHandshake(
 	return nil
 }
 
+// WriteUpgradeOpenChannel writes the agreed upon upgrade fields to the channel, sets the channel flush status to `NOTINFLUSH` and sets the channel state back to `OPEN`. This can be called in one of two cases:
```

Comment:
If I'm not mistaken, good doesn't support markdown syntax, so the backticks will not have the effect of formatting as code, unfortunately...

```suggestion
// WriteUpgradeOpenChannel writes the agreed upon upgrade fields to the channel, sets the channel flush status to NOTINFLUSH and sets the channel state back to OPEN. This can be called in one of two cases:
```

Reply:
It actually does but a super simplified version for some silly reason, can't find a specification on what it supports but backticks dont seem to be in the subset. 

p.s Doc links (i.e godoc creates an automatic link if you enclose things in brackets i.e `[Status.OPEN]`) could work here and would be nice to have but that's another subject.